src/audits/performance/speed-index-metric.js

```diff
@@ -17,7 +17,7 @@
 
 'use strict';
 
-const speedline = require('speedline');
+const speedline = /** @type {function(*)} */ (require('speedline'));
```

Comment:
can we do this with the `gulp-replace`/externs approach instead? That way it keeps it all in one place (`closure/`).